contracts/governance/ConfidentialERC20Votes.sol

```diff
@@ -223,20 +217,17 @@ abstract contract ConfidentialERC20Votes is IConfidentialERC20Votes, Confidentia
         _moveDelegates(currentDelegate, delegatee, delegatorBalance);
     }
 
-    function _getPriorVote(address account, uint256 blockNumber) internal view returns (euint64 votes) {
+    function _getPriorVote(address account, uint256 blockNumber) internal view returns (euint64) {
         uint32 nCheckpoints = numCheckpoints[account];
 
+        /* solhint-disable no-empty-blocks*/
         if (nCheckpoints == 0) {
             /// If there is no checkpoint for the `account`, return encrypted zero.
```

Comment:
i would remove this comment (since it returns an uninitialized handle now)

Reply:
You should also put an early explicit return here, since you are no longer using a named return. But I am not sure why you changed this convention? i see you are using named return in the other functions here. So maybe keep it?

Reply:
Otherwise now we have this warning when testing `Warning: Unnamed return variable can remain unassigned. Add an explicit return with value to all non-reverting code paths or name the variable.
   --> contracts/governance/ConfidentialERC20Votes.sol:220:89:
    |
220 |     function _getPriorVote(address account, uint256 blockNumber) internal view returns (euint64) {` obviously not optimal